astrometry/include/PixelMap.h

```diff
@@ -332,5 +334,37 @@ namespace astrometry {
     void operator=(const ColorTerm& rhs) = delete;
   
   };
+
+  class ASTMap: public PixelMap {
+    // Class representing a constant shift of position
```

Comment:
I don't understand why the ASTMap class is described as representing solely a constant shift.

Reply:
Just a mistake due to my using a different class as a template.